features/sp-solution-features/org.wso2.carbon.sp.solutions.apim.analytics.feature/pom.xml

```diff
@@ -0,0 +1,193 @@
+                                    <classpath>
+                                        <path refid="h2.classpath" />
+                                    </classpath>
+                                    <fileset file="${basedir}/src/main/resources/sql/h2.sql" />
+                                </sql>
+                                <echo message="##################### END ########################" />
+                            </tasks>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>org.wso2.carbon.maven</groupId>
+                <artifactId>carbon-feature-plugin</artifactId>
+                <version>${carbon.feature.plugin.version}</version>
+                <extensions>true</extensions>
+                <executions>
+                    <execution>
+                        <id>1-p2-feature-generation</id>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <propertyFile>../org.wso2.carbon.sp.solutions.apim.analytics.feature/etc/feature.properties</propertyFile>
+                            <adviceFileContents>
+                                <advice>
+                                    <name>org.wso2.carbon.p2.category.type</name>
+                                    <value>server</value>
+                                </advice>
+                                <advice>
+                                    <name>org.eclipse.equinox.p2.type.group</name>
+                                    <value>true</value>
+                                </advice>
+                            </adviceFileContents>
+                            <bundles>
+                                <bundle>
+                                    <symbolicName>org.wso2.analytics.apim.rest.api.file</symbolicName>
+                                    <version>${project.version}</version>
+                                </bundle>
+                                <bundle>
+                                    <symbolicName>org.wso2.sp.extension.siddhi.io.mgwfile</symbolicName>
+                                    <version>${project.version}</version>
+                                </bundle>
+                            </bundles>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+        <resources>
+            <resource>
+                <directory>${project.build.directory}/target</directory>
+            </resource>
+            <resource>
+                <directory>../org.wso2.carbon.sp.solutions.apim.analytics.feature/src/main/resources</directory>
+            </resource>
+        </resources>
+    </build>
+
+</project>
```

Comment:
Add new line EOF

Reply:
Fixed in a21aa920ceb5fdbd3f34fa9dddb75b5630876048 of https://github.com/wso2/analytics-apim.